client/src/components/app.jsx

```diff
@@ -15,9 +15,11 @@ class App extends React.Component {
     render() {
       console.log(this.state.eventUsers)
       return (
-        <div>
-        <div> Attendees {'(' + this.state.numberOfAttendees + ')'} </div>
-        <div> See All </div>
+        <div className="app">
+        <div className="attendees_header"> 
+        <div className="attendees_count"> Attendees {'(' + this.state.numberOfAttendees + ')'} </div>
+        <div className="seeAll"> See All </div>
+        </div>
```

Comment:
I'd suggest using tabs here to better reflect the DOM hierarchy.  Eg
<div className="app">
     <div className="attendees_header"> 
          <div className="attendees_count"> Attendees {'(' + this.state.numberOfAttendees + ')'} </div>
          <div className="seeAll"> See All </div>
     </div>
</div>